src/main/webapp/view/view.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import {intializeDiscussion} from '../view/discussion/discussion.js';
-import {loadTranscript} from './transcript/transcript.js';
+import {TranscriptArea} from './transcript/transcript-area.js';
```

Comment:
With export default you can remove the brackets from this import.

Reply:
gotcha!